tests/scalability/start_jmeter_ec2.sh

```diff
@@ -8,7 +8,7 @@ sed -i '2s~^~'MEDIC_URL=$MEDIC_URL'\n~' run_suite.sh
 sed -i '2s~^~'S3_PATH=s3://medic-e2e/scalability/$TAG_NAME-$GITHUB_RUN_ID'\n~' run_suite.sh
 sed -i '2s~^~'TAG_NAME=$TAG_NAME'\n~' run_suite.sh
 cat run_suite.sh
-# base64 encode our script and set as value for UserDAta in launch-specification
-jq --arg arn $SCALABILITY_ARN '.UserData = "'$(base64 run_suite.sh -w 0)'" | .IamInstanceProfile.Arn = $arn' launch-specification.json >> launch-specification-jmeter.json
-cat launch-specification-jmeter.json
-echo $(aws ec2 request-spot-instances --spot-price '0.333' --instance-count 1 --type 'one-time' --launch-specification file://launch-specification-jmeter.json --block-duration-minutes 60)
\ No newline at end of file
+
+echo Triggering EC2 Run Instance Command and getting Instance ID
+instanceID=$(aws ec2 run-instances --image-id ami-065ba2b6b298ed80f --instance-type c5.2xlarge --security-group-ids sg-0fa20cd785acec256 --block-device-mappings file://block-device-mapping.json --user-data file://run_suite.sh --instance-initiated-shutdown-behavior terminate | jq .Instances[0].InstanceId -r )
```

Comment:
The $SCALABILITY_ARN is set to be able to upload the jmeter results to S3. It looks like that is removed from here. We'll need that added back so we can review the results in S3. Once the instance is down it'll be destroyed and the results won't be around anymore right?

Reply:
Yea added the role to instance